test/client/query-subscribe.js

```diff
@@ -216,19 +216,28 @@ module.exports = function(options) {
         }
       ], function(err) {
         if (err) return done(err);
-        var query = connection.createSubscribeQuery('dogs', matchAllDbQuery, null, function(err) {
-          if (err) return done(err);
+        var query = connection.createSubscribeQuery('dogs', matchAllDbQuery);
+        query.on('error', done);
```

Comment:
If we only care about the first error from this test case, it may make sense to use `.once('error', done)` here as well